packages/iis/changelog.yml

```
@@ -1,9 +1,14 @@
 # newer versions go on top
+- version: "1.24.1"
+  changes:
+    - description: Fix Dashboard Links Panel by setting `useCurrentFilters` to `false`
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/14607
```

Can you fix this for the rest of the integrations as well?
```suggestion
      link: https://github.com/elastic/integrations/pull/14974
```